.gitignore

```
@@ -1,12 +1,25 @@
-.bundle/
+# Ignore bundler config.
+/.bundle
+/.nix-bundler
+
+# Ignore local environment files
+/.env*
```

I think we'd want to track .envrc if it exists, so just negate this one file:
```suggestion
/.env*
!/.envrc
```